object/object.go

```diff
@@ -17,22 +18,32 @@ type EncryptionKey struct {
 	entropy *[32]byte
 }
 
-// MarshalJSON implements the json.Marshaler interface.
-func (k EncryptionKey) MarshalJSON() ([]byte, error) {
-	return []byte(`"key:` + hex.EncodeToString(k.entropy[:]) + `"`), nil
+// String returns a hex-encoded representation of the key.
+func (k EncryptionKey) String() (s string) {
+	return "key:" + hex.EncodeToString(k.entropy[:])
 }
 
-// UnmarshalJSON implements the json.Unmarshaler interface.
-func (k *EncryptionKey) UnmarshalJSON(b []byte) error {
+// LoadString loads an EncryptionKey from a string.
+func (k *EncryptionKey) LoadString(s string) error {
 	k.entropy = new([32]byte)
-	if n, err := hex.Decode(k.entropy[:], bytes.TrimPrefix(bytes.Trim(b, `"`), []byte("key:"))); err != nil {
+	if n, err := hex.Decode(k.entropy[:], []byte(strings.TrimPrefix(s, "key:"))); err != nil {
 		return err
 	} else if n != len(k.entropy) {
 		return errors.New("wrong seed length")
 	}
 	return nil
 }
 
+// MarshalJSON implements the json.Marshaler interface.
+func (k EncryptionKey) MarshalJSON() ([]byte, error) {
+	return []byte(`"` + k.String() + `"`), nil
+}
+
+// UnmarshalJSON implements the json.Unmarshaler interface.
+func (k *EncryptionKey) UnmarshalJSON(b []byte) error {
+	return k.LoadString(string(bytes.Trim(b, `"`)))
+}
```

Comment:
if we implement `MarshalText` and `UnmarshalText`, JSON marshalling will automatically work *and* we can use those methods instead of `String`/`LoadString`.